net/interface.go

```diff
@@ -10,21 +9,14 @@ import (
 	"github.com/ericxtang/m3u8"
 	"github.com/golang/glog"
 	ffmpeg "github.com/livepeer/lpms/ffmpeg"
-	"github.com/livepeer/lpms/stream"
 )
 
 //VideoNetwork describes the interface for a Livepeer node network-layer library.
 type VideoNetwork interface {
```

Comment:
It's starting to feel like `VideoNetwork` and `VideoCache` has some similarities.  On a high level, I feel `VideoCache` is probably a misnomer for what it is today.  Maybe `VideoSource` is more appropriate.  If that's the case, maybe a `VideoNetwork` implements `VideoSource` + extra network-related methods like `GetNodeStatus`.

Reply:
+1 for `VideoSource`. We can flatten the VideoNetwork into VideoCache(Source) and see where that takes us.

Reply:
Flattening [here](https://github.com/livepeer/go-livepeer/pull/512/commits/2da29308070528f6f4c151292f4b264d26e0cfd3) and file rename [here](https://github.com/livepeer/go-livepeer/pull/512/commits/0a0e4e6f4093eb74550d1b538cfe072d79de1241)

Note that a side effect is we don't really a need for a VideoNetwork interface anymore, so that was removed.